test/Index/Store/record-include-locals.swift

```diff
@@ -0,0 +1,88 @@
+// RUN: rm -rf %t
```

Comment:
I'd probably be fine with just `driver-include-locals.swift` + the existing test in `test/Index/local.swift`. Could print jobs in that one as well and verify `-index-locals` is in the frontend command.

Reply:
Done (added checks to print the jobs, and kept the small example to verify that the frontend indeed emits the locals).